client/app/telecom/telephony/alias/configuration/changeType/telecom-telephony-alias-configuration-changeType.html

```diff
@@ -61,6 +61,18 @@ <h1 data-translate="telephony_alias_change_type_title"></h1>
                                 <option value="" data-translate="telephony_alias_config_change_type_label_select" disabled></option>
                         </select>
                     </div>
+
+                    <telephony-bulk-action data-service-type="alias"
+                                           data-billing-account="{{ AliasConfigurationChangeTypeCtrl.bulkDatas.billingAccount }}"
+                                           data-service-name="{{ AliasConfigurationChangeTypeCtrl.bulkDatas.serviceName }}"
```

Comment:
you can use `AliasConfigurationChangeTypeCtrl.number.serviceName` and remove it from `bulkDatas` object ;-)